source/tools/CustomHostCpuManager.h

```diff
@@ -0,0 +1,477 @@
+/*
+ * Copyright (C) 2016 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CUSTOM_HOST_CPU_MANAGER_H
+#define CUSTOM_HOST_CPU_MANAGER_H
+
+#include <cstdint>
+#include <iostream>
+#include <map>
+
+#include "HostTools.h"
+#include "scheddl.h"
+
+
+class CustomHostCpuManager : public HostCpuManagerBase
+{
+public:
+    /**
+     * Update runtime statistics and sleeps for wakeupTime_ns.
+     *
+     * @param wakeupTime_ns sleep duration, in nanoseconds
+     */
+    void sleepAndTuneCPU(int64_t wakeupTime_ns) override {
+        /* Read finishing time of the callback */
+        mEndingTime_ns = HostTools::getNanoTime();
+        getDeadlineAttribute(mDLEnd);
+
+        /* Update callback duration statistics */
+        calculateRuntime();
+
+        if (wakeupTime_ns == 0) {
+            /*
+             * If wakeupTime_ns is 0, it means that two consecutive bursts must
+             * be written, so the system is facing an overload condition. In
+             * this case, the bandwidth of the task is forced to a high value
+             * to solve the transitory.
+             */
+            boostBandwidth();
```

Comment:
The name "boost" could get confused with "schedtune.boost".

Reply:
I think this we can keep it.
Comments are explicit and the context should avoid any confusion.